lib/rest/Twilio.ts

```diff
@@ -0,0 +1,934 @@
+'use strict'
```

Comment:
Do we have a header to indicate that this is generated by openapi spec, similar to "this code is generated by yoyodyne"

Reply:
We don't currently do that for OAI-generated files, but would be a good standup/eng sync question